transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -0,0 +1,640 @@
+                }
+            });*/
+
+      let offerSdp = this.pc!.localDescription!;
+      // Filter audio codec
+      if (this.audioCodec && this.audioCodec !== "default") {
+        // @ts-ignore
+        offerSdp.sdp = this.sdpFilterCodec(
+          "audio",
+          this.audioCodec,
+          offerSdp.sdp,
+        );
+      }
+      // Filter video codec
+      if (this.videoCodec && this.videoCodec !== "default") {
+        // @ts-ignore
+        offerSdp.sdp = this.sdpFilterCodec(
+          "video",
+          this.videoCodec,
+          offerSdp.sdp,
+        );
+      }
+
+      logger.debug(`Will create offer for peerId: ${this.pc_id}`);
+
+      const url = `${this._options.params.baseUrl}${this._options.params.endpoints?.connect || ""}`;
+      // Send offer to server
+      const response = await fetch(url, {
+        body: JSON.stringify({
+          sdp: offerSdp.sdp,
+          type: offerSdp.type,
+          pc_id: this.pc_id,
+          restart_pc: recreatePeerConnection,
+        }),
+        headers: {
+          "Content-Type": "application/json",
+        },
+        method: "POST",
+      });
+
+      const answer: RTCSessionDescriptionInit = await response.json();
+      // @ts-ignore
+      this.pc_id = answer.pc_id;
+      // @ts-ignore
+      logger.debug(`Received answer for peer connection id ${answer.pc_id}`);
+      await this.pc!.setRemoteDescription(answer);
+      logger.debug(
+        `Remote candidate supports trickle ice: ${this.pc.canTrickleIceCandidates}`,
+      );
+    } catch (e) {
+      logger.debug(
+        `Reconnection attempt ${this.reconnectionAttempts} failed: ${e}`,
+      );
+      this.isReconnecting = false;
+      setTimeout(() => this.attemptReconnection(true), 2000);
+    }
+  }
+
+  private addInitialTransceivers() {
+    // Transceivers always appear in creation-order for both peers
+    // For now we are only considering that we are going to have 02 transceivers,
+    // one for audio and one for video
+    this.pc!.addTransceiver("audio", { direction: "sendrecv" });
+    this.pc!.addTransceiver("video", { direction: "sendrecv" });
+  }
+
+  private getAudioTransceiver() {
+    // Transceivers always appear in creation-order for both peers
+    // Look at addInitialTransceivers
+    return this.pc!.getTransceivers()[0];
+  }
+
+  private getVideoTransceiver() {
+    // Transceivers always appear in creation-order for both peers
+    // Look at addInitialTransceivers
+    return this.pc!.getTransceivers()[1];
+  }
+
+  private async startNewPeerConnection(
+    recreatePeerConnection: boolean = false,
+  ) {
+    this.pc = this.createPeerConnection();
+    this.addInitialTransceivers();
+    this.dc = this.createDataChannel("chat", { ordered: true });
+    await this.addUserMedia();
+    await this.negotiate(recreatePeerConnection);
+  }
+
+  private async addUserMedia(): Promise<void> {
+    logger.debug(`addUserMedias this.tracks(): ${this.tracks()}`);
```

Comment:
```suggestion
    logger.debug(`addUserMedia this.tracks(): ${this.tracks()}`);
```

same on lines 362 & 368

Reply:
Done.